package/rmt-server.spec

```diff
@@ -102,7 +102,7 @@ bundle.ruby2.5 install %{?jobs:--jobs %{jobs}} --without test development --depl
 mkdir -p %{buildroot}%{data_dir}
 mkdir -p %{buildroot}%{lib_dir}
 mkdir -p %{buildroot}%{app_dir}
-mkdir -p %{buildroot}%{conf_dir}ssl
+mkdir -p %{buildroot}%{conf_dir}/ssl
 mkdir -p %{buildroot}var/lib/rmt
```

Comment:
This creates `/home/abuild/rpmbuild/BUILDROOT/rmt-server-1.0.6-0.x86_64var/lib/rmt`.

Needs to be removed, there's already `mkdir -p %{buildroot}%{lib_dir}`, which creates `/home/abuild/rpmbuild/BUILDROOT/rmt-server-1.0.6-0.x86_64/var/lib/rmt`.